packages/venia-concept/src/actions/checkout/actions.js

```diff
@@ -20,7 +20,8 @@ const actionMap = {
     SHIPPING_METHOD: {
         SUBMIT: null,
         ACCEPT: null,
-        REJECT: null
+        REJECT: null,
+        INCORRECT_ADDRESS: null
```

Comment:
I'm a little bit concerned with how this will scale: if every action includes a full list of error scenarios, these will get out of hand fast. 

I'm thinking of these like exceptions or promise rejections: `INCORRECT_ADDRESS` is just one kind of `REJECT` scenario.

On the other hand, this likely does make things simpler to handle downstream.

Reply:
@supernova-at I've come to the same conclusion, thanks!